### PR TITLE
feat(BPopover): expose hide, show and toggle methods

### PR DESCRIPTION
**Pull request by duncansanderson**
# Describe the PR
Programmatically open, close and toggle `BPopover` component.

## Small replication

```
<BPopover
  ref="popover"
  :click="true"
   target="popover-button"
   title="popover"
>
  Hello
  <BButton @click="handleClose">Close</BButton>
</BPopover>
.
.
.
const popover = ref()
const handleClose = (event: Event) => popover.value.hideFn(event)
```

Solves #1242 

## PR checklist

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix :bug: - `fix(...)`
- [x] Feature - `feat(...)`
- [ ] ARIA accessibility - `fix(...)`
- [ ] Documentation update - `docs(...)`
- [ ] Other (please describe)

**The PR fulfills these requirements:**

- [x] Pull request title and all commits follow the [**Conventional Commits**](https://www.conventionalcommits.org/) convention or has an [**override**](https://github.com/googleapis/release-please#how-can-i-fix-release-notes) in this pull request body **This is very important, as the `CHANGELOG` is generated from these messages, and determines the next version type. Pull requests that do not follow conventional commits or do not have an override will be denied**

### CODE REVIEW

**Comment by duncansanderson** [comment]
@VividLemon I've updated BTooltip to extend the popover methods

File: packages/bootstrap-vue-next/src/components/BTooltip.vue [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
       <slot :name="name" v-bind="slotData" />
     </template>
   </BPopover>
 </template>
 
 <script setup lang="ts">
+import {ref} from 'vue'
 import BPopover from './BPopover.vue'
+
+const popover = ref()
```
Comment (VividLemon) [inline packages/bootstrap-vue-next/src/components/BTooltip.vue:13]:
```suggestion
const popover = ref<null | InstanceType<typeof BPopover>>(null)
```

File: packages/bootstrap-vue-next/src/components/BTooltip.vue [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
+ to allow the user to programmatically control the tooltip. */
+const hideFn = (event: Event) => popover.value.hideFn(event)
+const show = () => popover.value.show()
+const toggle = (event: Event) => popover.value.toggle(event)
+
+defineExpose({
+  hideFn,
+  show,
+  toggle,
+})
```
Comment (VividLemon) [inline packages/bootstrap-vue-next/src/components/BTooltip.vue:25]:
```suggestion
defineExpose({
  hide: popover.value.hide,
  show: popover.value.show,
  toggle: popover.value.toggle,
})
```

File: packages/bootstrap-vue-next/src/components/BPopover.vue [hunk 3]
```diff
@@ -432,6 +432,12 @@ const hideFn = (e: Event) => {
   }, delay)
 }
 
+defineExpose({
+  hideFn,
```
Comment (VividLemon) [inline packages/bootstrap-vue-next/src/components/BPopover.vue:436]:
```suggestion
  hide: hideFn,
```

**Review by VividLemon — CHANGES_REQUESTED** [review]

**Pull request merged**